tests/test_spark.py

```diff
@@ -249,6 +249,51 @@ def make_credential_secret(envvar, secret_path):
                     args=args)
 
 
+@pytest.mark.sanity
+@pytest.mark.smoke
+def test_kafka_cassandra_stream():
```

Comment:
Does this test actually involve Kafka? If not, the name is misleading.

Reply:
Not currently, the idea was that it would eventually. Maybe we'll remove this test for now.